plugin/core/collections.py

```diff
@@ -74,6 +75,19 @@ def remove(self, path: str) -> None:
             current = next_current
         current.pop(keys[-1], None)
 
+    def copy(self, path: Optional[str] = None) -> Any:
+        """
+        Get a copy of the value from the dictionary or copy of whole dictionary.
+
+        :param      path:  The path, e.g. foo.bar.baz, or None.
+
+        :returns:   A copy of the value stored at the path, or None if it doesn't exist.
+                    Note that this cannot distinguish between None values and
+                    paths that don't exist. If the path is None, returns a copy of the
+                    entire dictionary.
+        """
+        return deepcopy(self.get(path))
```

Comment:
Will this work OK if a value is involved? Say that the dict is `{"a": {"b" True}}` and we do `copy("a.b")` ?

Reply:
Yes:
```
>>> import copy; copy.deepcopy(True)
True
```